goreleaser.yaml

```diff
@@ -17,7 +17,7 @@ before:
 
 builds:
   - id: pomerium-cli
-    main: ./cmd/main.go
+    main: ./cmd
```

Comment:
I hit an issue with a root command actually working. See #11 ; maybe you found a way around it?

Reply:
It had to be built with `cmd/*.go`, which is what the syntax here does implicitly.